lib/taskana-core/src/main/java/pro/taskana/impl/AttachmentImpl.java

```diff
@@ -19,7 +19,8 @@
     private String taskId;
     private Timestamp created;
     private Timestamp modified;
-    private Classification classification;
+    private String classificationKey;
```

Comment:
beides brauchen wir ja eigentlich nicht, oder? Der Key is ja in der Summary enthalten...

Reply:
ClassificationSummary wird wie besprochen mit einem 2. Query geladen. Dieser nutzt die Domain von Task und den ClassificationKey, der hier direkt angegeben ist.
Im Model benötigen wir dieses Feld daher. In der DB ist nur der Key enthalten.